### PR TITLE
Add troubleshooting document, include initial examples - ledger connection, out-of-sync RevReg

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

**Comment by swcurran** [comment]
@WadeBarnes @ianco -- please take a look.  In a comment here, feel free to suggest other troubleshooting problems that we should include in this document.

Is the document a good idea?  Is the content sufficient? I couldn't think of where else to document the RevReg fixing stuff.

I believe we need an issue (and a PR) that is intended to prevent the "out of sync" scenario - correct?

File: Troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+Agreement). Once the error occurred, the RevReg state held by the ACA-Py agent,
+and the RevReg state on the ledger were different. Even after the ability to
+write to the ledger was restored, the RevReg could still not be published
+because of the differences in the RevReg state. Such a situation can now be
+corrected, as follows:
+
+To address this issue, some new endpoints were added to ACA-Py in Release 0.7.4,
+as follows:
+
+- GET /revocation/registry/<id>/issued - counts of the number of issued/revoked
```
Comment (WadeBarnes) [inline Troubleshooting.md:66]:
```suggestion
- GET `/revocation/registry/<id>/issued` - counts of the number of issued/revoked
```

File: Troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  within a registry
+- GET /revocation/registry/<id>/issued/details - details of all credentials
+  issued/revoked within a registry
+- GET /revocation/registry/<id>/issued/indy_recs - calculated rev_reg_delta from
```
Comment (WadeBarnes) [inline Troubleshooting.md:70]:
```suggestion
- GET `/revocation/registry/<id>/issued/indy_recs` - calculated rev_reg_delta from
```

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
Endpoint paths not displaying the `<id>` parameter.

File: Troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  the ledger
+  - This is used to compare ledger revoked vs wallet revoked credentials, which
+    is essentially the state of the RevReg on the ledger and in ACA-Py. Where
+    there is a difference, we have an error.
+- PUT /revocation/registry/<id>/fix-revocation-entry-state - publish an update
```
[the remark this answers is not in the document]
Comment (swcurran) [reply Troubleshooting.md]: Good catch -- thanks.  Fixed.

**Comment by swcurran** [comment]
@WadeBarnes -- please take another look.

**Review by WadeBarnes — APPROVED** [review]

**Pull request merged**